app/assets/stylesheets/modules/_nav.styl

```diff
@@ -26,7 +26,9 @@ nav.top-nav
     font-size 15px
     &:hover, &.active
       color $brand_primary
+      border none !important
```

Comment:
I'd like to avoid using `!important` as much as possible, as it can make further CSS refactoring and style changes more difficult.

Reply:
Let me remove it and push it again